test/functional/services/combo_box.ts

```diff
@@ -60,26 +75,38 @@ export function ComboBoxProvider({ getService }) {
       await this.closeOptionsList(comboBoxElement);
     }
 
-    async filterOptionsList(comboBoxSelector, filterValue) {
-      log.debug(`comboBox.filterOptionsList, comboBoxSelector: ${comboBoxSelector}, filter: ${filterValue}`);
+    /**
+     * filtering combobox items by value
+     *
+     * @param {string} comboBoxSelector data-test-subj selector
+     * @param {string} filterValue
+     * @memberof ComboBox
+     */
+    public async filterOptionsList(comboBoxSelector: string, filterValue: string): Promise<void> {
+      log.debug(
+        `comboBox.filterOptionsList, comboBoxSelector: ${comboBoxSelector}, filter: ${filterValue}`
+      );
       const comboBox = await testSubjects.find(comboBoxSelector);
       await this._filterOptionsList(comboBox, filterValue);
       await this.closeOptionsList(comboBox);
     }
 
-    async _filterOptionsList(comboBoxElement, filterValue) {
+    private async _filterOptionsList(
+      comboBoxElement: WebElementWrapper,
+      filterValue: string
+    ): Promise<void> {
       const input = await comboBoxElement.findByTagName('input');
       await input.clearValue();
       await this._waitForOptionsListLoading(comboBoxElement);
       await input.type(filterValue);
       await this._waitForOptionsListLoading(comboBoxElement);
     }
 
-    async _waitForOptionsListLoading(comboBoxElement) {
+    private async _waitForOptionsListLoading(comboBoxElement: WebElementWrapper): Promise<void> {
```

Comment:
maybe we can remove `_` since its now in `ts`

Reply:
it can break up ci

Reply:
Not sure what you mean @vitalics, that method is only referenced in this class and assuming you rename it everywhere it's used in this file it shouldn't cause any problems.